### ISSUE TITLE
Blocked <URL> from calling getURL on its opener.

### ISSUE THREAD

**Issue by parkerlreed** [jiahaog/nativefier#463]
### Description

Trying to package up a Salesforce login. Package works and I can browse around but I have a window that opens externally (thanks to --internal-urls ".*") that needs to check what called it.

### Steps to reproduce issue

Open popup window that needs to check what called it

### Details

- Are you nativefying a *public* website? - Not public as in for testing.
- Full nativefier command used to build your app: `nativefier --insecure --internal-urls ".*" --verbose "https://test.salesforce.com/"`
- Version of Nativefier (run `nativefier --version`): `v7.4.1`
- Version of node.js (run `node --version`): `v8.6.0`
- OS: `Arch Linux`
- Error message / stack trace (if any):

```
Blocked https://<url>/Auth/ExternalAuthentication/CheckFrame from calling getURL on its opener.
```

**Comment by ronjouch** [comment]
@parkerlreed I don't understand, please clarify/reformulate.

**Comment by parkerlreed** [comment]
I haven't been able to dig anything else up on my end. Was just testing anyways. I'll close out for now. Thanks.

**Comment by welchste** [comment]
Hello, is it possible to re-open this issue?
I have used this awesome tool to make a Linux native app for RingCentral's Glip service. Everything works except for when you try to initiate a video call from the app. Clicking the video link will direct the browser to open an external application for another application installed on my PC. I'm getting the same url blocked error as the OP.
```
Blocked https://app.glip.com/ from calling getURL on its opener.
```
Here is how I'm configuring the app with Nativefier if it helps.
```
nativefier --name 'RingCentral' --app-version '0.3' --icon RingCentral-icon.png --counter --ignore-gpu-blacklist --internal-urls "(.*login\.ringcentral.com.*|.*glip.com)" --inject ./discord-dark-theme-for-glip.user.js --single-instance --tray https://app.glip.com
```

**Comment by welchste** [comment]
One more bit of detail, I also ran the internal urls to inclide (.*ringcentral.com) with the same issue. As the link will redirect to meetings.ringcentral.com

**Comment by ronjouch** [comment]
@welchste is this still a problem in 8.0.4 ?

**Comment by welchste** [comment]
Hey @ronjouch it failed but with a different output.
```
Blocked https://app.glip.com/ from accessing guestId: 2
```
Maybe I need to find a correct argument setting for this?

**Comment by ronjouch** [comment]
> Hey @ronjouch it failed but with a different output: `Blocked https://app.glip.com/ from accessing guestId: 2`. Maybe I need to find a correct argument setting for this?

@welchste nah, it's a legit Nativefier bug, we're wiring our windows wrong and [Electron complains](https://github.com/electron/electron/pull/21512/files?w=1). Needs a bugfix. Thanks for the re-test 🙂.

Patch welcome, or maybe I'll look at it someday if I need it.

**Comment by welchste** [comment]
An app launches, much like a zoom link in your browser opening up the native zoom client. Same exact scenario.
